src/fr/articles/landmarks-aria/index.njk

```diff
@@ -2,136 +2,207 @@
+        […]
+      &lt;/div&gt;
 
+      &lt;div role="contentinfo"&gt;
+        &lt;p&gt;…contenu du pied de page&nbsp;: copyright…&lt;/p&gt;
+        […]
+      &lt;/div&gt;
+    &lt;/body&gt;
+    </code></pre>
+  </li>
+</ol>
+
+<h3 class="mt-4 mb-0" id="equivalence">Correspance entre les balises HTML5 et les régions (<i lang="en">landmarks</i>) ARIA</h3>
+  <table class="table mt-2 mb-4">
+      <caption class="visually-hidden" aria-labelledby="equivalence"></caption>
+      <thead>
+        <tr>
+          <th class="table-column1-width">Balises HTML5</th>
+          <th>Régions <abbr>ARIA</abbr></th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><code>header</code></td>
+          <td><code>role="banner"</code> seulement si placé dans une balise <code>body</code></td>
+        </tr>
+        <tr>
+          <td><code>nav</code></td>
+          <td><code>role="navigation"</code></td>
+        </tr>
+        <tr>
+          <td><code>--</code></td>
+          <td><code>role="search"</code></td>
+        </tr>
+          <td><code>main</code></td>
+          <td><code>role="main"</code></td>
+        </tr>
+          <tr>
+          <td><code>form</code></td>
+          <td><code>role="form"</code> s'il y a un nom accessible donné par aria-labelledby, aria-label ou title</td>
+        </tr>
+        <tr>
+          <td><code>section</code></td>
+          <td><code>role="region"</code> s'il y a un nom accessible donné par aria-labelledby, aria-label ou title</code> ou <code>aria-label</code></td>
+        </tr>
+        <tr>
+          <td><code>aside</code></td>
+          <td><code>role="complementary"</code></td>
+        </tr>
+        <tr>
+          <td><code>footer</code></td>
+          <td><code>role="contentinfo"</code> seulement si placé dans une balise <code>body</code></td>
+        </tr>
+      </tbody>
+  </table>
+
+<h3 class="mt-4">Usages</h3>
+<ul class="usages">
+  <li>La balise <code>header</code> ou l'attribut de région <abbr>ARIA</abbr> <code>role="header"</code> permet d'identifier l’en-tête de la page. A n'utiliser qu'une seule fois dans la page.</li>
+  <li>La balise <code>main</code> ou l'attribut <code>role="main"</code> identifie le contenu principal de la page. A n'utiliser qu'une seule fois dans la page.</li> 
+  <li>La balise <code>aside</code> ou l'attribut <code>role="complementary"</code> est à utiliser pour identifier du contenu annexe au contenu principal de la page (information complémentaire, publicité…).</li>
+  <li>La balise <code>footer</code> ou l'attribut <code>role="contentinfo"</code> permet d’identifier le pied de page. A n'utiliser qu'une seule fois dans la page.</li>
+</ul>
+<p>Ces 4 balises ou attributs de région (<i lang="en">landmarks</i>) <abbr>ARIA</abbr> doivent être placés directement dans une balise <code>body</code> (et pas dans une autre balise elle-même imbriquée dans la balise <code>body</code>).</p>
```

Comment:
It may be good to add some space here in order to make it easier to understand that this sentence is linked to previous list and not the next one.